src/lib.rs

```diff
@@ -279,6 +293,7 @@ impl<'a> EnvironmentBuilder<'a> {
     ///   It will be simply passed to `LoadLibraryW`.
     ///
     /// Default value: `WebView2Loader.dll`.
+    #[cfg(not(feature = "embed-dll"))]
```

Comment:
Can you update the documentation of this function?

The documentation of this whole module (at the start of the source file) also mentions the handling of the DLL file, it need to updated too.

Reply:
It could be very confusing that a feature makes a function unavailable.

Maybe we don't need the feature flag at all. Just use the embeded DLL by default, and load an external DLL if this function is called.

Reply:
The problem is that if the features are no longer used the executable is bigger for nothing.

Reply:
With LTO, dead code elimination should be able to eliminate the DLL content if it is never used.

Reply:
In the last commit i review the logic …
if the "embed-dll" :
 * We use the embedded dll if there is no dll_file_path defined
 * To use a specific dll we must use with_dll_file_path
else :
 * it's like before